### DISCUSSION TITLE
Optolith vs. Optolith Insider?

### DISCUSSION THREAD

**Discussion by milkpirate** [elyukai/optolith-client#1358] [Q&A] [answered]
## Source(s)

![image](https://user-images.githubusercontent.com/9251021/156078852-42a5c8aa-f3d1-42a4-8a78-2aa44db9540d.png)
https://www.ulisses-ebooks.de/product/209711/Optolith-Heldenverwaltung&language=de

## Describe the ambiguity
Just curious, wahts the difference between the two?

**Comment by elyukai** [comment, marked as answer]
Sorry for the late reply, haven't seen it! 😬🙈

The "Insider" is the development version of Optolith, which represents the lasted (running) build. It includes content from unfinished publications, but may of course be unstable. The progress of unfinished is documented in issues in this repository (a good starting point may be https://github.com/elyukai/optolith-client/projects/4), so even though e.g. special abilities can be bought, it doesn't mean their logic is done or there may be breaking changes until their implementation is done and they've been released in a stable version.

Hope this helps!
